pallet/src/lib.rs

```diff
@@ -1,15 +1,39 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+
+use codec::{Decode, Encode};
 use frame_support::Parameter;
 use sp_std::prelude::*;
 
+use frame_support::{sp_runtime::BoundToRuntimeAppPublic, traits::OneSessionHandler};
 pub use pallet::*;
 
 #[frame_support::pallet]
 pub mod pallet {
     use super::*;
-    use frame_support::{pallet_prelude::*, sp_runtime::RuntimeAppPublic, sp_std};
+    use frame_support::{
+        pallet_prelude::*,
+        sp_runtime::{DigestItem, RuntimeAppPublic},
+        sp_std,
+    };
     use frame_system::pallet_prelude::*;
+    use primitives::{AuthoritiesLog, ALEPH_ENGINE_ID};
+
+    #[derive(Encode, Decode)]
+    pub struct SessionChange<T>
+    where
+        T: Config,
+    {
+        /// The block number the session was created.
+        pub created_at: T::BlockNumber,
+        pub session_id: u64,
+        pub changed: bool,
```

Comment:
Here the name `changed` is out of the context and the name is not descriptive enough. Add a comment or change the name.

Reply:
I have changed it to `authorities_changed`.